packages/apps/lightning-ui-docs/.storybook/addons/components/NumberRow.js

```diff
@@ -18,22 +18,33 @@
 
 import React, { useState } from 'react';
 import { NumberControl } from '@storybook/blocks';
+import { updateComponentValue } from './componentUtils';
 import TableRow from './TableRow';
 
-export default function NumberRow({ label, defaultValue, scope, onChange }) {
-  const [value, setValue] = useState(defaultValue);
+/**
+ * @returns a style row with a number control
+ */
+
+export default function NumberRow({
+  version,
+  styleProp,
+  defaultValue,
+  componentName,
+  updateGlobals
+}) {
+  const [fieldValue, setValueState] = useState(defaultValue);
+
   return (
     <TableRow
-      label={label}
-      scope={scope}
+      label={styleProp}
       control={
         <NumberControl
-          name={label}
-          value={value}
-          min={0} // may need to revisit this if there are ever valid negative values
+          name={styleProp}
+          key={`Number-${componentName}-${styleProp}-${version}`}
```

Comment:
I'm not sure `key` is needed here, since each NumberRow renders a single NumberControl. When rendering an array of NumberRow components, having a unique key on each of those should be enough to tell React to re-render a NumberRow component if the key changes on it. (It looks like that's already being done in ComponentStylesContent, so think just this line can be removed).